library/ecp_curves.c

```diff
@@ -4737,6 +4737,8 @@ int mbedtls_ecp_group_load( mbedtls_ecp_group *grp, mbedtls_ecp_group_id id )
     ECP_VALIDATE_RET( grp != NULL );
     mbedtls_ecp_group_free( grp );
 
```

Comment:
Please explain to me how it can be right to free the `grp` and then init it right after that?
https://os.mbed.com/teams/sandbox/code/mbedtls/docs/tip/ecp_8c_source.html#l00321

Clearly shows everything gets freed after that.
What am I missing here?

As far as I can understand this will essentially set the already freed memory region to zero.